src/main/java/fr/foodlimit/api/user/User.java

```diff
@@ -1,7 +1,9 @@
 package fr.foodlimit.api.user;
 
+import lombok.Data;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
+@Data
 public class User {
 
   private String name;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "name" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)